docs/testlayer.rst

```diff
@@ -76,24 +76,29 @@ ZCMLLayer
 
 We now want a layer that loads up some ZCML from a file. The default
 is ``ftesting.zcml``, but here we'll load a test ``testlayer.zcml``.
+We can also choose to provide extra ZCML features (here, "devmode").
```

Comment:
This probably should either explain what `devmode` does, or provide a link to the docs which do.